Orm/Xtensive.Orm/Orm/Configuration/DomainConfiguration.cs

```diff
@@ -732,7 +742,7 @@ protected override void CopyFrom(ConfigurationBase source)
       ignoreRules = (IgnoreRuleCollection) configuration.IgnoreRules.Clone();
       shareStorageSchemaOverNodes = configuration.ShareStorageSchemaOverNodes;
       versioningConvention = (VersioningConvention) configuration.VersioningConvention.Clone();
-      
+      taggingBehavior = configuration.taggingBehavior;
```

Comment:
Use property to be inline with already existing code.

Reply:
done